test/fixtures/linter/autofix/bigquery/003_templating/after.sql

```diff
@@ -21,9 +21,9 @@ FROM
         {{action}}_raw_effect_sizes
         {% else %}
         JOIN
-                    {{action}}_raw_effect_sizes
+            {{action}}_raw_effect_sizes
             USING
-                            ({{corr_states}})
-{% endif %}
+                ({{corr_states}})
+    {% endif %}
```

Comment:
Good fix - with this fix, we indent better than we could before so remove some incorrect indents that shouldn't be there.

Reply:
Interesting! My recent L003 PR also had an impact to this area of this file. I hope the two sets of fixes co-exist or complement one another nicely. Very cool to see improvements in these old real-world test cases. (This particular file is from an old project at my employer.)

Reply:
Seeing some failures now so looks like they might be clashing 😢

Looking...

Reply:
Argh. Let me know if I can help. IIRC, my PR has code that calls this code.

Reply:
No my code is just rubbish - comparing to identical source strings for equality instead of source to templated 😔 trying to fix now.

Reply:
Reverted so resolving.